aspnetcore/signalr/hubs.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Use Hubs in ASP.NET Core SignalR
+author: rachelappel
+description: Learn how to use hubs in ASP.NET Core SignalR.
+manager: wpickett
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 03/23/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/hubs
+---
+
+# Use Hubs in SignalR for ASP.NET Core
+
+By [Rachel Appel](https://twitter.com/rachelappel) and [Kevin Griffin](http://twitter.com/1kevgriff)
+
+## What is a SignalR Hub
+
+The SignalR Hubs API enables you to call methods on connected clients from the server. In server code, you define methods that are called by client. In client code, you define methods that are called from the server. SignalR takes care of everything behind the scenes that makes real-time client-to-server and server-to-client communications possible.
+
+## Configure SignalR hubs
+
+Since SignalR is middleware, a call to `services.AddSignalR` is required in the `ConfigureServices` method of the `Startup` class to register the service.
+
+[!code-javascript[Configure service](hubs/sample/startup.cs?range=35)]
+
+When adding SignalR functionality to an ASP.NET MVC application, setup SignalR routes by calling `app.UseSignalR` in the `Configure` method before the other routes.
+
+[!code-javascript[Configure routes to hubs](hubs/sample/startup.cs?range=55-58)]
+
+## Create and use hubs
+
+To create a hub, inherit from the `Hub` class, and add public methods to it. Clients can call methods that are defined as `public`. Methods that use access modifiers such as `private` or `protected` can't accept calls from clients.
```

Comment:
>To create a hub, inherit from the `Hub` class, and add public methods to it.

"inherit" as a verb doesn't really work well here to me. I think something like "Create a hub by declaring a class that inherits from `Hub`, and add public methods to it" might work better.

> Methods that use access modifiers such as `private` or `protected` can't accept calls from clients.

This sentence seem redundant with the previous one